src/parser/jobs/mnk/modules/Steppies.tsx

```diff
@@ -53,8 +55,13 @@ export default class Steppies extends Module {
 	}
 
 	private onDamage(event: NormalisedDamageEvent): void {
-		if (event.hitCount > 0) {
-			const boot = new Boot(event.criticalHits > 0, this.combatants.selected.hasStatus(STATUSES.LEADEN_FIST.id), event.timestamp)
+		if (event.hits > 0) {
```

Comment:
TS error - this needs to be event.hitCount due to the signature/naming changes for NormalisedDamageEvent from #794